### PR TITLE
Minor additions to bug-fixes by Dan

### PR DESCRIPTION
**Pull request by robertavram**

### CODE REVIEW

File: src_ts/components/pages/assessments/assessment-tab-pages/questionnaire/assessment-questionnaire-page.ts [hunk 1]
```diff
@@ -15,6 +15,7 @@ import {fireEvent} from '../../../../utils/fire-custom-event';
 import {formatServerErrorAsText} from '../../../../utils/ajax-error-parser';
 import {SharedStylesLit} from '../../../../styles/shared-styles-lit';
 import '../../../../common/layout/etools-error-warn-box';
+import { green } from 'chalk';
```
Comment (adi130987) [inline src_ts/components/pages/assessments/assessment-tab-pages/questionnaire/assessment-questionnaire-page.ts]: Do we need this?

File: src_ts/components/pages/assessments/assessment-tab-pages/questionnaire/assessment-questionnaire-page.ts [hunk 2]
```diff
@@ -36,11 +37,19 @@ class AssessmentQuestionnairePage extends connect(store)(LitElement) {
           margin-bottom: 32px;
           font-size: 24px;
           color: white;
-          background-color: var(--primary-shade-of-green);
           box-shadow: 0 2px 2px 0 rgba(0, 0, 0, 0.14),
                     0 1px 5px 0 rgba(0, 0, 0, 0.12),
                     0 3px 1px -2px rgba(0, 0, 0, 0.2);
         }
+        .red {
+          background-color: var(--primary-shade-of-red);
```
Comment (adi130987) [inline src_ts/components/pages/assessments/assessment-tab-pages/questionnaire/assessment-questionnaire-page.ts]: We might want to reuse color from `app-theme` and not add new ones

File: src_ts/components/pages/assessments/assessment-tab-pages/questionnaire/assessment-questionnaire-page.ts [hunk 3]
```diff
@@ -101,31 +110,17 @@ class AssessmentQuestionnairePage extends connect(store)(LitElement) {
...
   setOverallRatingDisplay(overall_rating: {rating: number, display: string}) {
-    if (overall_rating && overall_rating.display !== 'Unknown') {
+    if (overall_rating && overall_rating.display !== '-') {
       this.overallRatingDisplay = overall_rating.display;
     } else {
       this.overallRatingDisplay = '';
     }
   }
 
-  noQuestionnaireHtml() {
```
Comment (adi130987) [inline src_ts/components/pages/assessments/assessment-tab-pages/questionnaire/assessment-questionnaire-page.ts]: Why is this removed?

Comment (robertavram) [reply src_ts/components/pages/assessments/assessment-tab-pages/questionnaire/assessment-questionnaire-page.ts]: This should never be a real world scenario.. and if it is.. it's a vaild bug. we already have a lot of code.. trying to cut dow on anything that's not necessary
